frontend/model/contracts/chatroom.js

```diff
@@ -178,7 +180,8 @@ sbp('chelonia/defineContract', {
       sideEffect ({ contractID }) {
         Vue.set(sbp('state/vuex/state').chatRoomUnread, contractID, {
           readUntil: undefined,
-          mentions: []
+          mentions: [],
+          others: []
```

Comment:
added `others` field here, as we now want the app to know the 'unread' state of `MESSAGE_TYPES.INTERACTIVE` as well. We might also need to use this for some other types in the future too e.g. `MESSAGE_TYPE.POLL`)